browser/download/bubble/download_display_controller_unittest.cc

```diff
@@ -333,7 +333,7 @@ class DownloadDisplayControllerTest : public testing::Test {
     Browser::CreateParams params(profile_, true);
     params.type = Browser::TYPE_NORMAL;
     params.window = window.release();
-    browser_ = std::unique_ptr<Browser>(Browser::Create(params));
+    browser_ = Browser::DeprecatedCreateOwnedForTesting(params);
```

Comment:
is there a follow-up issue for these to test without Browser or convert to browser test?

Reply:
to be clear I don't mean a follow-up issue for chromium rebase team, just for tracking purposes